delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/indexing/CompositeSinks.scala

```diff
@@ -0,0 +1,63 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.indexing
+
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.client.BlazegraphClient
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.indexing.BlazegraphSink
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.CompositeSink
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.config.CompositeViewsConfig
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.indexing.CompositeViewDef.ActiveViewDef
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.model.CompositeViewProjection
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.model.CompositeViewProjection.{ElasticSearchProjection, SparqlProjection}
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.client.ElasticSearchClient
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.RemoteContextResolution
+import ch.epfl.bluebrain.nexus.delta.sdk.model.BaseUri
+import ch.epfl.bluebrain.nexus.delta.sourcing.stream.Operation.Sink
+
+/**
+  * Defines the sinks for the indexing progress for a composite view
+  */
+trait CompositeSinks {
```

Comment:
I will create a ticket to address that in a later PR if that is ok.
Indeed, it does feel we could refactor this part.

Reply:
Works for me. In the ticket perhaps mention that we should review the `CompositeSink` documentation so that we don't forget about that part. This way I will merge #4138 soon without changing the name and we do that when we get to it.